sed/binning/binning.py

```diff
@@ -267,7 +267,7 @@ def bin_dataframe(
             Defaults to None.
         pbar (bool, optional): Option to show the tqdm progress bar. Defaults to True.
         n_cores (int, optional): Number of CPU cores to use for parallelization.
-            Defaults to all but one of the available cores. Defaults to N_CPU-1.
+            Defaults to all but one of the available cores.
```

Comment:
why not use n_cores from config in this module?

Reply:
Effectively it is being used if called from the processor, but the binning functions themselves are not within classes and don't use a config.